localtools/src/integrationTest/kotlin/testconfiguration/containers/instances/ManagedProvenanceTestContainer.kt

```diff
@@ -63,8 +63,8 @@ class ManagedProvenanceTestContainer : ManagedTestContainer<ProvenanceTestContai
                     // - Copy asset_classification_smart_contract.wasm from the artifacts directory that is created from that command
                     // - Paste the wasm into localtools/integrationTest/resources
                     // - Uncomment the following line, and comment out the other wasmLocation line:
-                    // wasmLocation = ContractWasmLocation.LocalFile.ProjectResource("asset_classification_smart_contract.wasm"),
-                    wasmLocation = ContractWasmLocation.GitHub(contractReleaseTag = "v${AppResources.CONTRACT_VERSION}"),
+                    wasmLocation = ContractWasmLocation.LocalFile.ProjectResource("asset_classification_smart_contract.wasm"),
```

Comment:
TODO: Before merging this, the contract PR needs to be merged.  Then, this line can go back to normal and the locally-committed smart contract in this PR can be deleted.